test/matchmaker/player.test.js

```diff
@@ -13,13 +14,18 @@ const {
   tournamentJSON
 } = require("./player.test.data");
 
+const {
+  testMatches
+} = require("./match.test.data");
+
 const EXPECTED_PLAYER = {
   id: "PLAYER1",
   firstname: "Gerhard",
   lastname: "Acker",
   clubname: "ESV SF Neuaubing",
   gamesWon: 0,
   matchIds: [],
+  opponentIds: [],
```

Comment:
ein player hat nun ja das Attribut opponentIDs....dieses muss jetzt überall manuell hinzugefügt werden korrekt?

Reply:
in den TestDaten, ja, ansonsten ist das nur die Stelle an der der Spieler erzeugt wird